build.assets/Dockerfile

```diff
@@ -272,19 +272,22 @@ RUN helm plugin install https://github.com/vbehar/helm3-unittest && \
     HELM_PLUGINS=/home/ci/.local/share/helm/plugins helm plugin list
 
 # Install JS gRPC tools.
-RUN (npm install --global grpc_tools_node_protoc_ts@5.0.1 grpc-tools@1.12.4)
+ARG NODE_GRPC_TOOLS_VERSION
+ARG NODE_PROTOC_TS_VERSION
+RUN npm install --global "grpc-tools@$NODE_GRPC_TOOLS_VERSION" "grpc_tools_node_protoc_ts@$NODE_PROTOC_TS_VERSION"
 
-# Install protobuf and grpc build tools.
+# Install protoc.
 ARG PROTOC_VER
+RUN VERSION="$PROTOC_VER" && \
+  PB_REL='https://github.com/protocolbuffers/protobuf/releases' && \
+  cd /tmp && \
+  curl -fsSLO "$PB_REL/download/v$VERSION/protoc-$VERSION-linux-$(uname -m).zip"  && \
+  unzip protoc*.zip -d /usr/local && \
+  rm -f protoc*.zip
```

Comment:
uhh I think /tmp is one if the directories we mount from the host, what if I have some other protoc.zip files in there?

Reply:
We don't mount /tmp for babybox, only the project directory.

Reply:
but this is in the main dockerfile, no?

Reply:
maybe I'm misunderstanding, I'm not a docker pro

Reply:
You are right, true. I think this is the current behavior, though, I just made the snippet smaller.

https://github.com/gravitational/teleport/blob/1296b06955fcc9d2d071aa07bafc8d5c8ba3d853/build.assets/Dockerfile#L281-L286

Reply:
Maybe I'm being paranoid but the wildcard in `unzip protoc*.zip -d /usr/local` seems slightly sketchier than the existing `unzip /tmp/${PROTOC_TARBALL} -d /usr/local`. Could possibly unzip some unnecessary (or evil!) stuff or delete some extra stuff.

Reply:
or more likely I'm building two buildboxes on different versions concurrently and the `rm -f protoc*.zip` tramples the other one

Reply:
Fixed.